src/main/kotlin/no/nav/amt/person/service/person/dbo/PersonUpsert.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.amt.person.service.person.dbo
+
+import no.nav.amt.person.service.person.model.IdentType
+import java.util.*
+
+data class PersonUpsert(
```

Comment:
Trenger vi et eget objekt for upsert? Er ikke dette likt som Person-objektet?

Reply:
Det er likt Person ja 😅 Vi kan godt bruke det for min del 👍